recipes/fmt/all/test_package/conanfile.py

```diff
@@ -1,19 +1,35 @@
 import os
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain
+try:
+    from conan.tools.build import cross_building as tools_cross_building
```

Comment:
This is at `conan.tools.build` since `1.46`, is it necessary to do this double import?

Reply:
it was failing to import in some previous versions, I need to re-check if it's still necessary

Reply:
anyway, it's no replaced by `can_run`, so it's irrelevant